docs/reference/api/resources/command.rst

```diff
@@ -73,6 +73,14 @@ as well. It isn't possible to have functionality exposed on a toolbar that
 isn't also exposed by the app. So, ``cmd2`` will be added to the app, even though
 it wasn't explicitly added to the app commands.
 
+If you want to change the action that a command executes after it's been created, use:
+
+.. code-block:: python
+
+    ...
+
+    cmd1.action = changed_callback
+
```

Comment:
This documentation isn't needed. There's a defined property; basic python idiom dictates that it can be set.
```suggestion
```